pkg/extension/workspace/example/envoy/template/defaults.go

```diff
@@ -41,7 +41,7 @@ var (
 					SocketAddress: &envoycore.SocketAddress{
 						Address: "127.0.0.1",
 						PortSpecifier: &envoycore.SocketAddress_PortValue{
-							PortValue: 9901,
+							PortValue: 9901, // TODO: why default admin port to 9901?
```

Comment:
no idea on this one..

Reply:
Envoy folks usually uses `9901` for admin, but it's just a tradition I think.